legend-engine-xt-relationalStore-memsql-execution-tests/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/memsql/test/MemSQLContainer.java

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.memsql.test;
+
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+public class MemSQLContainer extends JdbcDatabaseContainer<MemSQLContainer>
+{
+    public static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("memsql/cluster-in-a-box");
```

Comment:
Should we add the registry to the image name, and also include the ability to override it?